packages/dm-core-plugins/src/table/TableRow/TableRow.tsx

```diff
@@ -128,6 +114,17 @@ export function TableRow(props: TTableRow) {
             </Table.Cell>
           )
         })}
+        {config.functionality?.openAsTab && (
+          <Table.Cell style={{ textAlign: 'center' }}>
+            <Button
+              title="Open"
+              variant="ghost"
+              onClick={() => openItemAsTab(item, index)}
+            >
+              <Icon data={open_in_browser} aria-hidden /> Open
+            </Button>
```

Comment:
There is several differences here relative to the Table.Cell for openAsExpanable (ie cell style, button variant, button disabled, title, aria-hidden, tooltip...). Is there a specific reason for all those differences?

Reply:
disabled is definitely missing and I'll take a look, see what makes sense for each button and do some improvements